app/models/manageiq/providers/azure/cloud_manager/provision.rb

```diff
@@ -0,0 +1,5 @@
+class ManageIQ::Providers::Azure::CloudManager::Provision < ManageIQ::Providers::CloudManager::Provision
+  include_concern 'Cloning'
+  include_concern 'StateMachine'
+  include_concern 'Configuration'
```

Comment:
@bronaghs I don't see a `provision/configuration.rb` file.  That's probably why your Travis build is failing.